mitodl/ol-infrastructure
# config: enable daily digest (MITx Online)

### What are the relevant tickets?

https://github.com/mitodl/hq/issues/3881

### Description (What does it do?)
- Enables the option to subscribe to daily forum digest in MITx Online

### Screenshots (if appropriate):
Take a look at the screenshots in the associated ticket

### How can this be tested?
Once deployed, We should be able to see the enable digest option under discussions
